composer.json

```diff
@@ -80,9 +78,90 @@
         "install-ext": "make -f DDMakefile install_ini install SUDO=sudo $*",
         "test-ext": "make -f DDMakefile test_c",
         "test": "phpunit --colors=always --configuration=phpunit.xml",
+        "test-all-56": [
+            "@test-unit",
+            "@test-integration",
+            "@test-integrations-56"
+        ],
+        "test-all-70": [
+            "@test-unit",
+            "@test-integration",
+            "@test-integrations-70"
+        ],
+        "test-all-71": [
+            "@test-unit",
+            "@test-integration",
+            "@test-integrations-71"
+        ],
+        "test-all-72": [
+            "@test-unit",
+            "@test-integration",
+            "@test-integrations-72"
+        ],
         "test-integration": [
             "echo \"Integration tests require the agent up and running. Use composer run-agent.\"",
-            "phpunit --colors=always --configuration=phpunit.xml --testsuite=integration"
+            "@composer test -- --testsuite=integration"
+        ],
+        "test-integrations-56": [
+            "@composer scenario:update",
```

Comment:
Are these lists of commands going to be the same across each integration?

Reply:
Right now only mongo tests are different, I expect that as we add more and more versions (soon) they will diverge quite a bit.

Reply:
:+1: